docs/getting-started/overview.md

```diff
@@ -0,0 +1,42 @@
+# Overview
+
+This is an overview of how agents communicate with each other as compared to regular client/server http connections. 
+
+# Http communication protocol
+
+In most applications, the client communicates with the server using http protocol where a request is made to the server and the client awaits the response from the server with a specific timeout after which a timeout exception is raised if we didn’t get any response from server.
+
+![http request](../images/dcd1.png)
+
+# Agent to agent communication
+
+On the other hand, agents communicate using [didcomm](https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0005-didcomm) communication protocol, while this protocols have much more to talk about but the most important concern here is how the communication flow goes. For the sake of demonstration, lets assume 2 agents want to communicate Agent Alice and Agent Bob.
+
+1- Agent Alice will send a connection request to agent Bob either directly or through a mediator (another routing agent) using outbound transporter
+2- Agent Bob receives the message (through inbound transporter) and process the message 3- Agent Bob sends the response in a new request (using outbound TP) sent back to agent Alice
+4- Agent Alice receives the message through the in-bound TP
+5- Agent Alice process the message (under the hood through Aries) and raises an event with attached data relevant to communication context
+6- The controller (your app) receives the data using a subscribed callback to the corresponding event types and act upon it
```

Comment:
This doesn't render very nicely. I'd suggest to use markdown numbered lists

1. ...
2. addd

Reply:
Yes @TimoGlastra I am working on more documentation and guides
And yes the list is formatted